notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>Maybe add something like: "If we know that a manifold is conveniently described as an open set, then some of the manifold's abstract methods can be rewritten in a specific form. Note that we do not (yet) specify <em>which</em> manifold we are talking about (whether it is a sphere or another surface), we are just saying that we are looking at some manifold that can be described as an open set."</p><p> </p><p>remove : we have not talked about tangent vectors yet, so don't worry if you don't understand what <code>is_tangent</code> and <code>to_tangent</code> do yet. We only mentioned them here for completeness.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>done</p>